main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -2294,11 +2297,6 @@ protected Void doInBackground(final Void... params) {
                 DataStore.saveCache(cache, EnumSet.of(SaveFlag.DB));
                 return null;
             }
-
-            @Override
-            protected void onPostExecute(final Void v) {
-                notifyDataSetChanged();
-            }
```

Comment:
But data (in this case personal note is updated without calling this).  The only difference I found after changing back is the "note has been saved" info which is not shown any longer  :( I'll fix this and come back...